### PR TITLE
Fix typo in integration_test.rst

### PR DESCRIPTION
**Pull request by aciba90**
## Proposed Commit Message

```
Fix typo in integration_test.rst
```

## Checklist:

 - [x] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/topics/contributing.html)
 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: doc/rtd/topics/integration_tests.rst [hunk 1]
```diff
@@ -54,8 +54,8 @@ or by providing an environment variable of the same name prepended with
     CLOUD_INIT_PLATFORM='ec2' pytest tests/integration_tests/
 
 
-Cloud Interation
-================
+Cloud Integration
```
Comment (TheRealFalcon) [inline doc/rtd/topics/integration_tests.rst:57]:
```suggestion
Cloud Interaction
```

**Review by TheRealFalcon — CHANGES_REQUESTED** [review]
Nice find on the typo, but it's actually supposed to be 'Interaction', not 'Integration'

**Comment by aciba90** [comment]
> Nice find on the typo, but it's actually supposed to be 'Interaction', not 'Integration'

Sorry, I typed without thinking too much and usually cloud and integration go hand by hand. 🤦🏼

**Review by TheRealFalcon — APPROVED** [review]
Thanks!

**Pull request merged**